backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/OngoingRepositoryOperation.java

```diff
@@ -0,0 +1,5 @@
+package com.virtuslab.gitmachete.backend.api;
+
+public enum OngoingRepositoryOperation {
+  NO_OPERATION, CHERRY_PICKING, MERGING, REBASING, REVERTING, APPLYING, BISECTING
```

Comment:
It is rather a philosophical question but... how should the enums be ordered? Lexicographically or by "usage frequency"? :thinking: 

Let it be anyway

Reply:
Hmm, it's a good question. Actually, in this case I'd prefer `NO_OPERATION` first as it's something special and then other actions in lexical order.
I will change this in that way.